src/pads/turfs.rs

```diff
@@ -0,0 +1,46 @@
+use std::ptr::null_mut;
+use auxtools::raw_types::values::{Value, ValueData, ValueTag};
+use crate::pads::{byond_imports, find_by_call};
+
+byond_imports!(
+    #[cfg(windows)]
+    fn GET_STEP: extern "cdecl" fn(*mut Value, Value, u8) -> *mut Value
+        = find_by_call!(
+            windows => "8d 85 d8 f8 ff ff 50 8d 89 78 03 00 00 e8 >?? ?? ?? ?? ff 70 04 ff 30"
+        );
+    #[cfg(unix)]
+    fn GET_STEP: extern "cdecl" fn(*mut Value, u32, Value, u8) -> *mut Value
```

Comment:
In fact, that `*mut Value` here is used for structure-return only, right? 
Then you can just declare type as `fn(u32, Value, u8) -> Value`

Reply:
Note: Crashes on Linux, probably bc of signature mismatch, might be due to the fact it is structure return, and compiler ABI differs for `fn(*mut Value, u32, Value, u8) -> *mut Value` and `fn(u32, Value, u8) -> Value`